controller/todo_controller.go

```diff
@@ -0,0 +1,118 @@
+package controller
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/wdonet/golang-bootcamp-2020/data/handler"
+	"github.com/wdonet/golang-bootcamp-2020/domain/model"
+	"github.com/wdonet/golang-bootcamp-2020/utilities"
+)
+
+// GetTodos process GET on /todos
+func GetTodos(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	todos := handler.GetTodosFromFile()
+	json.NewEncoder(w).Encode(todos)
+}
+
+// GetTodo process GET on /todos/{id}
+func GetTodo(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	params := mux.Vars(r)
+	todos := handler.GetTodosFromFile()
+	found := utilities.FindTodoByID(params["id"], todos)
+	if found == nil {
+		found = &model.Todo{}
+	}
```

Comment:
Another example of the error handling would be that here maybe we would like to change the `200 OK` response for something like a `404 Not Found`.